src/components/Timeline/updates/TimelineAssigned.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormattedMessage } from 'react-intl';
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import PersonHoverCard from 'components/PersonHoverCard';
+import TimelineActor from '../TimelineActor';
+import { ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
+import { ZetkinUpdateAssignee } from 'types/updates';
+
+interface Props {
+  update: ZetkinUpdateAssignee;
+}
+
+const TimelineAssigned: React.FunctionComponent<Props> = ({ update }) => {
+  return (
+    <Box>
+      <Grid alignItems="center" container direction="row" spacing={2}>
+        <Grid item>
+          <TimelineActor actor={update.actor} size={32} />
+        </Grid>
+        <Grid item>
+          <Grid container direction="row" spacing={1}>
+            {renderDescriptionText()}
+          </Grid>
+        </Grid>
+      </Grid>
+    </Box>
+  );
+
+  // Forms the update description, which consists of an "action" (i.e. what the update changed),
+  function renderDescriptionText() {
+    return (
+      <>
+        <Typography component={Grid} item variant="body2">
+          {renderActionText()}
+        </Typography>
+        <Typography color="textSecondary" component={Grid} item variant="body2">
+          <ZetkinRelativeTime datetime={update.timestamp} />
+        </Typography>
+      </>
+    );
+  }
```

Comment:
Personally I feel like all these render functions can just go inline in the JSX. I don't see the benefit of breaking them out.

Reply:
And I believe we must have room for personal tastes in how to write code...